mne/io/brainvision/brainvision.py

```diff
@@ -79,30 +80,49 @@ def __init__(self, vhdr_fname, montage=None,
                  event_id=None, verbose=None):  # noqa: D102
         # Channel info and events
         logger.info('Extracting parameters from %s...' % vhdr_fname)
-        vhdr_fname = os.path.abspath(vhdr_fname)
         info, fmt, self._order, mrk_fname, montage = _get_vhdr_info(
             vhdr_fname, eog, misc, scale, montage)
         events = _read_vmrk_events(mrk_fname, event_id, response_trig_shift)
         _check_update_montage(info, montage)
         with open(info['filename'], 'rb') as f:
-            f.seek(0, os.SEEK_END)
-            n_samples = f.tell()
-        dtype_bytes = _fmt_byte_dict[fmt]
+            if isinstance(fmt, dict):  # ASCII, this will be slow :(
```

Comment:
*Maybe* (not important) give a warning for this case? ("ASCII detected, loading data will be slow")